azkaban-db/src/main/sql/create.containerization-tables-all.sql

```diff
@@ -120,3 +120,20 @@ CREATE UNIQUE INDEX version_set_md5
 -- alter table execution_flows add column dispatch_method TINYINT default 1;
 -- CREATE INDEX ex_flows_dispatch_method ON execution_flows (dispatch_method);
 
+-- Definition for hp_flow_owners table. It contains list of owners who have
+-- access to add/remove high priority flows.
+CREATE TABLE IF NOT EXISTS hp_flow_owners (
+    owner            VARCHAR(64)     NOT NULL PRIMARY KEY,
```

Comment:
I saw insert and select query where this column name is "name". This sql script and code in PR will break things. Can you please test this part and change column name appropriately?

Reply:
Good catch. I tested it myself but I am admin so it never went to user query.

Reply:
Please test everything as user on solo-server and verify all the APIs. You will miss many bugs otherwise.